src/styles/index.scss

```diff
@@ -1,3 +1,124 @@
-body {
+$main-text-color: #060B35;
+$second-text-color: #616070;
+$button-color: #00ACDC;
+$back-color: #FFF;
+
+* {
   margin: 0;
+  box-sizing: border-box;
+  font-family: Roboto, sans-serif;
+}
+
+.card {
+  display: flex;
+  box-sizing: border-box;
+  flex-direction: column;
+  width: 200px;
+  padding: 32px 16px 17px;
+  border-radius: 5px;
+  background-color: $back-color;
+  border: 1px solid #F3F3F3;
+
+  color: $main-text-color;
+
+  &__img {
+    height: 134px;
+    width: 160px;
+    align-self: center;
+  }
+
+  &__product-title {
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    padding: 40px 0 4px;
+  }
+
+  &__product-code {
+    color: $second-text-color;
+    font-size: 10px;
+    font-weight: 400;
+    line-height: 14px;
+  }
+
+  &__review {
+    padding: 16px 0 24px;
+    display: flex;
+    justify-content: space-between;
+
+    &-reviews {
+      display: flex;
+      align-items: center;
+      color: $main-text-color;
+      font-size: 10px;
+      font-weight: 400;
+      line-height: 14px;
+    }
+  }
+
+  &__price {
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    padding-bottom: 16px;
+
+    &-text {
+      color: $second-text-color;
+      font-size: 12px;
+      font-weight: 400;
+      line-height: 18px;
+    }
+
+    &-sum {
+      color: $main-text-color;
+      font-size: 16px;
+      font-weight: 700;
+      line-height: 18px;
+    }
+  }
+
+  &__button {
+    width: 166px;
+    height: 40px;
+    border-radius: 5px;
+    background-color: $button-color;
+
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    text-decoration: none;
+    color: $back-color;
+    font-size: 14px;
+    font-weight: 700;
+    line-height: 16px;
+    text-transform: uppercase;
+
+    &:hover {
+      border: 1px solid $button-color;
+      background-color: $back-color;
+      color: $button-color;
+    }
+  }
+}
+
+.stars {
+  display: flex;
+  list-style: none;
+  padding-inline-start: 0;
+
+  &__active-star {
+    height: 16px;
+    width: 16px;
+    background-image: url(/images/star-active.svg);
+    background-repeat: no-repeat;
+    background-position: center;
+  }
+
+  &__unactive-star {
+    height: 16px;
+    width: 16px;
+    background-image: url(/images/star.svg);
+    background-repeat: no-repeat;
+    background-position: center;
+  }
```

Comment:
It`s wrong, according to the previous task, the stars block should have a modifier for the block where you can set how much stars will be active.